packages/engine/Source/Shaders/Builtin/Functions/getDynamicAtmosphereLightDirection.glsl

```diff
@@ -0,0 +1,23 @@
+/**
+ * Select which direction vector to use for dynamic atmosphere lighting based on the czm_atmosphereDynamicLighting enum.
+ *
+ * @name czm_getDynamicAtmosphereLightDirection
+ * @glslfunction
+ * @see DynamicAtmosphereLightingType.js
+ *
+ * @param {vec3} positionWC the position of the vertex/fragment in world coordinates. This is normalized and returned when dynamic lighting is turned off.
+ * @return {vec3} The normalized light direction vector. Depending on the enum value, it is either positionWC, czm_lightDirectionWC or czm_sunDirectionWC
+ */
+vec3 czm_getDynamicAtmosphereLightDirection(vec3 positionWC) {
```

Comment:
Add a note in #11681 that this can be used in the SkyAtmosphereShader (and perhaps the globe shader if we are okay with switching to a uniform rather than a define.